wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallController/CallController.swift

```diff
@@ -119,6 +118,22 @@ final class CallController: NSObject {
             return false
         }
     }
+
+    private func acceptDegradedCall(conversation: ZMConversation) {
+        guard let userSession = ZMUserSession.shared() else { return }
```

Comment:
Do we not have `enqueue` methods on `UserSession`?

Reply:
we do have them, but userSession is not a property of CallController, here

Reply:
I think it's being injected in the initialized, but maybe not retained.

Reply:
ok I looked and I'd rather not touch this here, if I do then yes it works for this function but for `cancelCall` I need ZMUserSession so downcasting the userSession again...

and we need userSession to call methods for callkit apparently (cf callNotificationStyle...)